### ISSUE TITLE
BTM - Apply the new architecture flow

### ISSUE THREAD

**Issue by PazYanoverr** [amplication/amplication#7864]
The following should happen when the user chooses to apply the new architecture in this modal - 
![Image](https://github.com/amplication/amplication/assets/149934977/e32637cb-2caf-466d-bdef-2aa4d6de1baf)
1.  In the "same" modal / dialog,-
     1. Our loader should be displayed.
     2. The headline should be "Processing your new architecture"
     3. The text should be "Please wait as we handle the creation of your new architectural design."
     4. The design should be like - ![Image](https://github.com/amplication/amplication/assets/149934977/52cfc53c-216f-46dc-ac30-72acac415215)
2. When the process completes successfully, the following should be displayed (in the "same" modal)-      1.  Headline- "Your new architecture is ready!"
      2. Text- "What should we do next?" 
      3. Display 3 options- (1) "Generate the code for my new architecture", (2) "Show my updated project overview", (3) "Display my new architecture" 
      4. The design should be like- <img width="1115" alt="image" src="https://github.com/amplication/amplication/assets/149934977/dfb2a76a-0242-4903-881c-b78cef577c8b">
3. When the process fails- an error message should appear saying that "We encountered a problem while processing your new architecture. Please try again in a few minutes"
4. Check also https://github.com/amplication/amplication/issues/7819, as the design may overlap (and should be implemented only once...)

**Comment by PazYanoverr** [comment]
@yuval-hazaz @morhag90 @mulygottlieb - Please review the ticket description, and share your comments

**Comment by overbit** [comment]
# Test results: ❌ 

https://jam.dev/c/e40ed159-3bca-4495-a108-3789288f154c

1. Apply changes doesn't match the requirements  ![Image](https://github.com/amplication/amplication/assets/2861984/b3ba7914-c315-462f-bc85-e71916cccf9e)
2. Step 1.ii is not displayed correctly as it should show "Processing your new architecture" ![Image](https://github.com/amplication/amplication/assets/2861984/a02b4b6e-97d2-4a2c-8f7a-af288ee06ea9)

**Comment by PazYanoverr** [comment]
@morhag90 @mulygottlieb  - A few comments / questions
1. After creating a new service it appears like it's not connected to git, although the project is connected-
<img width="1237" alt="image" src="https://github.com/amplication/amplication/assets/149934977/b9d23af1-e9fa-431a-a591-4d9ea3df0f80">
<img width="1244" alt="image" src="https://github.com/amplication/amplication/assets/149934977/43cdf003-3608-489e-b37f-cfb18cd1574b">
2. In case a new service is created- any reason not to allow REST and admin UI?
Besides that- it looks good, and passed my tests

**Comment by morhag90** [comment]
@PazYanoverr  

1. Can you please add the project configuration?
2. That was the original request (create without admin-ui; look at the original screen options).

**Comment by PazYanoverr** [comment]
1. See one of the screenshots above
2. This is just the UI that Alex created. I think (and IIRC, it was discussed in the spec) that we should allow REST, GQL, and admin UI unless there are some technical issues.

**Comment by yuval-hazaz** [comment]
I agree that all 3 should be selected by default.
Or, we can copy the configuration of the source service

**Comment by PazYanoverr** [comment]
@yuval-hazaz - We can copy the source configuration, but I think that at this point, we should keep it as simple as possible.

**Comment by mulygottlieb** [comment]
@PazYanoverr, please open a new ticket for creating the new services with all 3 options (REST/GQL/Admin UI) by default.

**Comment by mulygottlieb** [comment]
QA results: ✅
Based on the updated requirements (found in Notion and Figma), the current UI is the expected one; therefore, this passed QA.
See the attached video, which shows the full flow (UX is as intended).


https://github.com/amplication/amplication/assets/1912933/2ae207d2-ca45-432d-9a21-223c1e83eea9

**Comment by PazYanoverr** [comment]
@mulygottlieb - https://github.com/amplication/amplication/issues/7968
